target/linux/layerscape/patches-5.4/802-can-0025-can-flexcan-add-LPSR-mode-support-for-i.MX7D.patch

```diff
@@ -47,8 +47,7 @@ Signed-off-by: Joakim Zhang <qiangqing.zhang@nxp.com>
 +			err = pm_runtime_force_suspend(device);
  			if (err)
  				return err;
- 
--			err = pm_runtime_force_suspend(device);
++
 +			pinctrl_pm_select_sleep_state(device);
```

Comment:
For layerscape kernel patches, all look fine to me except this one.
I think the two functions should not be needed here any more.

pm_runtime_force_suspend(device);
pinctrl_pm_select_sleep_state(device);

The pinctrl_pm_select_sleep_state(device) may be put to right place instead.
I attached the patch I rebased. Please review.